dipy/align/parzenhist.pyx

```diff
@@ -789,10 +789,10 @@ cdef _compute_pdfs_sparse(double[:] sval, double[:] mval, double smin,
         cnp.npy_intp offset, valid_points
         cnp.npy_intp i, r, c
         double rn, cn
-        double val, spline_arg, sum
+        double val, spline_arg, s
```

Comment:
same comment as above: We try to avoid 1 letter variable (apart from maths equation). Can you rename it? `total, summation, ...`

Reply:
Done in 4e9cab5.